src/rail/estimation/algos/deepdisc.py

```diff
@@ -167,44 +199,46 @@ def run(self):
         # optimizer = return_optimizer(cfg)
         optimizer = solver.build_optimizer(cfg_loader, model)
 
-        """
-        When using the single test dictionary, add this code and replace "mapper" below
-        
-        
+
+        #When using the single test dictionary, add this code and replace "mapper" below
         def dc2_key_mapper(dataset_dict):
             filename = dataset_dict["filename"]
             return filename
 
         IR = DC2ImageReader()
         mapper = RedshiftDictMapper(IR, dc2_key_mapper).map_data
-        """
+        
+
+        # mapper = RedshiftFlatDictMapper().map_data
 
-        mapper = RedshiftFlatDictMapper().map_data
         loader = d2data.build_detection_train_loader(
-            train_data, mapper=mapper, total_batch_size=batch_size
+            metadata, mapper=mapper, total_batch_size=batch_size
         )
-        test_loader = d2data.build_detection_test_loader(
-            train_data, mapper=mapper, batch_size=batch_size
+
+        eval_loader = d2data.build_detection_test_loader(
+            metadata, mapper=mapper, batch_size=batch_size
         )
 
         saveHook = return_savehook(output_name)
-        lossHook = return_evallosshook(val_per, model, test_loader)
+        lossHook = return_evallosshook(val_per, model, eval_loader)
         schedulerHook = return_schedulerhook(optimizer)
         hookList = [lossHook, schedulerHook, saveHook]
 
         trainer = return_lazy_trainer(
             model, loader, optimizer, cfg, cfg_loader, hookList
         )
 
+        # how often the trainer prints something to screen, could be a config
```

Comment:
Make this `5` a global variable like `NUM_EPICS_BETWEEN_EPOCHS=5` (name TBD)

Reply:
This should be an input config parameter

Reply:
Tracked in https://github.com/lincc-frameworks/deepdisc/issues/75

Reply:
Referenced in issue #23